docs/preview/02-Features/publishing-events.md

```diff
@@ -1,54 +1,121 @@
----
+﻿---
 title: "Publishing Events"
 layout: default
 ---
 
-## Publishing Events
+# Azure Event Grid publishing
+We provide support for publishing Azure EventGrid, CloudEvents and custom events to a custom Azure Event Grid topics. 
+This event publishing via a dedicated `IEventGridPublisher` implementation that can either be added to the dependency container of your application, or can be created directly via our `EventGridPublisherBuilder`.
+
+- [Azure Event Grid publishing](#azure-event-grid-publishing)
+  - [Usage](#usage)
+  - [Publishing events](#publishing-events)
+    - [Publishing EventGridEvents](#publishing-eventgridevents)
+    - [Publishing CloudEvents](#publishing-cloudevents)
+  - [Resilient Publishing](#resilient-publishing)
+    - [Exponential retry](#exponential-retry)
+    - [Circuit breaker](#circuit-breaker)
+    - [Combination](#combination)
 
-We provide support for publishing custom events to a custom Azure Event Grid Topics.
+## Usage
+Adding simple Azure EventGrid publishing to your application only required the following registration.
+> ⚠ Note that this way of registering requires the [Arcus secrect store](https://security.arcus-azure.net/features/secret-store) to retrieve the necessary authentication secrets to interact with the Azure EventGrid topic.
+
+```csharp
+using Microsoft.Extensions.DependencyInjection;
+
+public void ConfigureServices(IServiceCollection services)
+{
+    // Requires Arcus secret store, for more information, see: https://security.arcus-azure.net/features/secret-store
+    services.AddSecretStore(stores => ...);
+
+    // Registers an `IEventGridPublisher` to your application to a custom topic.
+    services.AddEventGridPublishing(
+        "https://my-eventgrid-topic-endpoint", 
+        "<my-authentication-secret-name>");
+}
+```
 
-Import the following namespace into your project:
+To create an `IEventGridPublisher` instance directly via the `EventGridPublisherBuilder`, the authentication key of your custom topic endpoint is required. Since the authentication key is passed directly, this approach doesn't require the Arcus secret store.
```

Comment:
```suggestion
To create an `IEventGridPublisher` instance directly via the `EventGridPublisherBuilder`, the authentication key of your custom topic endpoint is required. Since the authentication key is passed directly, this approach doesn't require the Arcus secret store.  Nevertheless, it is advised to safely store your authentication key in a secret store.
```

Reply:
Yes, great! 👍